### DISCUSSION TITLE
[Release Vote] Vote for Release v0.6.0-rc2

### DISCUSSION THREAD

**Discussion by shidayang** [apache/amoro#2216] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Amoro 0.6.0 release.

The commit ID is https://github.com/NetEase/amoro/commit/8d50d9c1ce6d48ba5d14bce9cd40bbeb8bd5d34b

This corresponds to the tag: v0.6.0-rc2
https://github.com/NetEase/amoro/commits/v0.6.0-rc2
https://github.com/NetEase/amoro/tree/v0.6.0-rc2
Please check out, build, and test.

Please vote in the next 72 hours and add your test steps when voting.

**Comment by shidayang** [comment]
+1

I have built and deployed AMS locally with jdk8, hadoop2

Tested the following features:
Register catalog(include paimon, iceberg)
Table list and detail(include paimon, iceberg)
Terminal(include paimon, iceberg)
All about optimizing iceberg table

**Comment by wangtaohz** [comment]
+1.

I have built and deployed AMS locally with jdk8, hadoop3, flink-1.15.

Tested the following features:

- using derby as sysdb
- create the optimizer group (local container) and start a local optimizer
- create internal catalog (Iceberg format, both Hadoop and S3 storage) and Glue catalog (Iceberg format, S3 storage)
- spark insert overwrite/flink cdc (Iceberg format)
- table optimizing and result

**Comment by baiyangtx** [comment]
-1 

For bug  https://github.com/NetEase/amoro/pull/2229

**Comment by xxubai** [comment]
+1
I tested with following conditions:
- Deployed on AWS: Glue(with DynamoDB lock) and S3
- Flink external optimizer
- TiDB as sysdb
- self-optimizing
- enable data-expiration

**Comment by xieyi888** [comment]
+1
I have built and deployed AMS locally with jdk8, hadoop2

- create flink external optimizer
- create mix_hive and mix_iceberg table for parquet/orc fileformat
- read/write mix_hive and mix_iceberg  table for parquet/orc  fileformat with spark/flink engine
- table optimizing and result
